app/services/analytics_service.rb

```diff
@@ -34,6 +37,20 @@ def track_user_sign_out
     )
   end
 
+  def track_workshop_view(name, url)
```

Comment:
Might `workshop_path` be a better variable name than `name`? The logic you have below, `name.humanize.titleize` might not make sense if you pass the workshop's name instead of path (ex. "Personal Website").